hexrd/ui/calibration_crystal_editor.py

```diff
@@ -155,28 +157,15 @@ def position(self, v):
             blocker = QSignalBlocker(w)  # noqa: F841
             w.setValue(v[i])
 
-    @property
-    def matrix_scaling(self):
-        scaling = np.array([np.sqrt(2)] * 9)
-        diagonal_indices = [0, 4, 8]
-        for i in diagonal_indices:
-            scaling[i] = 1
-
-        return scaling
-
     @property
     def inverse_stretch(self):
-        m = (self.stretch_matrix * self.matrix_scaling).reshape(3, 3)
-        m = np.linalg.inv(m).flatten()
-        indices = [0, 4, 8, 5, 2, 1]
-        return [m[i] for i in indices]
+        m = np.array(self.stretch_matrix).reshape(3, 3)
+        return matrixutil.symmToVecMV(np.linalg.inv(m), scale=True)
 
     @inverse_stretch.setter
     def inverse_stretch(self, v):
-        indices = [0, 5, 4, 5, 1, 3, 4, 3, 2]
-        m = np.array([v[i] for i in indices]).reshape(3, 3)
-        m = np.linalg.inv(m).flatten()
-        self.stretch_matrix = m / self.matrix_scaling
+        m = matrixutil.vecMVToSymm(v, scale=True)
```

Comment:
Have to be careful about what we are using in this setter.  If the 6-vector is the the same as the values that go into the grain parameters, then the scaling is already in there and the "scale" kwarg should be False.  What is the use case you were envisioning?

Reply:
This setter for `inverse_stretch` is supposed to be the opposite of the getter for `inverse_stretch`. This setter is where we take the 6-vector and convert it into the raw matrix that gets displayed in the GUI.

So, the `inverse_stretch` getter converts the raw matrix into scaled 6-vector.
The `inverse_stretch` setter converts the scaled 6-vector into the raw matrix.